bin/tx-prover/src/commands/mod.rs

```diff
@@ -42,6 +42,8 @@ pub struct ProxyConfig {
     pub available_workers_polling_time_ms: u64,
     /// Health check interval in seconds.
     pub health_check_interval_secs: u64,
+    /// Prometheus metrics port.
+    pub prometheus_port: u16,
```

Comment:
Related to one of the previous questions: would it make sense to make this setting optional?

Reply:
It has a default value in the configuration file, in case that this is not set the proxy service will fail to start. Maybe we can make it optional the case that we want don't want to expose the metrics but I think that we can stop collecting it (maybe adding a feature needed above each metrics update? though it sounds terrible).

Reply:
Based on my current understanding, I think it is fine to keep it as mandatory.